### ISSUE TITLE
Tampermonkey update breaking weather loading from forecast.io

### ISSUE THREAD

**Issue by gitguys** [Tampermonkey/tampermonkey#923]
## Expected Behavior

Basic javascript userscript loads and shows weather in an iframe from forecast.io - it's worked for years with Tampermonkey on all browsers including Chrome, Safari and Firefox Mac and Windows.

## Actual Behavior

Disappears with latest Tampermonkey 4.10.6112 update or perhaps the 4.9.x version.  The weather refuses to load on all browsers including Chrome and Firefox on Mac and Windows.

In Chrome I see an error that says:

![requestsblocked](https://user-images.githubusercontent.com/11050768/80855791-3927ff80-8c01-11ea-912e-7aaa66b8b11f.jpg)

**"Requests to the server have been blocked by an extension"**

Because of this issue I'm not updating Tampermonkey on Safari, but I assume its probably broke there too.

Is there a way to revert back to the previous version of Tampermonkey?  This really stinks.

## Specifications

Chrome and Firefox on Mac and Windows.  Probably Safari too I assume.

## Script

```js

// ==UserScript==
// @name                Weather iFrame
// @namespace      wat
// @version             0.1
// @description       try to take over the world!
// @include             https://www.startpage.com/en/?
// @include             https://www.startpage.com/
// @include             https://www.startpage.com/en/*
// @include             https://www.google.com/
// @include             https://www.google.com/webhp*
// @require             http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js

// ==/UserScript==

var iframe=document.createElement("iframe");
iframe.id="weather"
iframe.class="weather"
iframe.src = '//forecast.io/embed/#lat=39.636077&lon=-105.039545&name=Denver&color=white'
iframe.setAttribute("style", "padding: 10px; ;border:5px;height: 220px; width: 380px;  border-radius: 40px; position:absolute; top: -45px; left: -10px !important;");
document.documentElement.appendChild(iframe);

(function() {
    'use strict';

    var stylesheet = `

#weather {
    opacity: 0.6; /* css standard */
    filter: contrast(50%) brightness(100%); /* internet explorer */
    transition: 2s;
}

#weather:hover {
    opacity: 1; /* css standard */
    filter: contrast(50%) brightness(200%); /* internet explorer */
}

div#hptl {display: none !important}

`;

    var styleElem = document.createElement('style');
    styleElem.textContent = stylesheet;
    document.head.appendChild(styleElem);
})();
```

**Comment by derjanb** [comment]
This is a CSP limitation of the "outer" page:

```Refused to frame 'https://forecast.io/' because it violates the following Content Security Policy directive: "frame-src 'self' *.startpage.com".```

I'll add a new API to handle this: https://github.com/Tampermonkey/tampermonkey/issues/881#issuecomment-620801057

Duplicate of #881
